target/linux/ath79/dts/qca9531_comfast_cf-e313ac.dts

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-e313ac", "qca,qca9531";
+	model = "COMFAST CF-E313AC";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &wlan;
+		led-failsafe = &wlan;
+		led-upgrade = &wlan;
```

Comment:
Use rssihigh for that or remove signalling, but do not use LAN/WLAN leds:
https://github.com/openwrt/openwrt/commit/abbbecaa73dcb5a36932a224f74488d0c5e3b3cd

Reply:
Seems legit.